### PR TITLE
Create Resource “introduction-to-design-systems”

### PR DESCRIPTION
**Pull request by bonnieAcameron**
Automatically generated by Netlify CMS

Previews: 
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cms/resources/introduction-to-design-systems/resources/#21st-century-idea 
and 
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cms/resources/introduction-to-design-systems/resources/introduction-to-design-systems/

### CODE REVIEW

**Review by mejiaj — CHANGES_REQUESTED** [review]
@bonnieAcameron looks good overall! I think the `source_url` is breaking the preview, so we can remove that.

I've also added some small formatting changes. I tested for functionality and typos.

File: content/resources/introduction-to-design-systems.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+date: 2023-02-10 14:30:00 -0500
+# Originally published at the following URL
+source_url: https://digital.gov/resources/intro-to-design-systems
```
[the remark this answers is not in the document]
Comment (mejiaj) [reply content/resources/introduction-to-design-systems.md:4]: @bonnieAcameron removing lines `3-4` should do it.

[the remark this answers is not in the document]
Comment (bonnieAcameron) [reply content/resources/introduction-to-design-systems.md]: added the commit!

**Review by mejiaj — APPROVED** [review]
Looks good, thanks for making those changes @bonnieAcameron.

This is ready for your review @ToniBonittoGSA.

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**